dev/clang-format/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Usage:
+#
+# $ docker build . -t librepcb/clang-format:6
+#
+# Then from the main directory:
+#
+# $ docker build . -t librepcb/clang-format:6
+# $ docker run --rm -t -i -v $(pwd):/code librepcb/clang-format:6 /bin/bash -c "cd /code && dev/format_code.sh"
+FROM ubuntu:bionic
+
+RUN apt-get -q update \
+ && apt-get -qy install \
+  clang-format \
```

Comment:
This does not guarantee that version 6 gets installed, it would be better to install `clang-format-6.0` and set it as default with `update-alternatives`.

Reply:
It does, as long as the distribution is ubuntu bionic. But I can also use update-alternatives if you want.